Test cases for 1-my_list.py
-------------------------------------------

>>> MyList = __import__('1-my_list').MyList
>>>
>>> my_list = MyList()
>>>
>>> my_list.append(1)
>>> my_list.append(4)
>>> my_list.append(2)
>>> my_list.append(3)
>>> my_list.append(5)
>>> print(my_list)
[1, 4, 2, 3, 5]
>>> my_list.print_sorted()
[1, 2, 3, 4, 5]
>>> li1 = [5, 2, 9, 1, 7]
>>> my_list = MyList(li1)
>>> my_list.print_sorted()
[1, 2, 5, 7, 9]
>>> li2 = []
>>> my_list = MyList(li2)
>>> my_list.print_sorted()
[]
>>> my_list = MyList()
>>> my_list.append(1)
>>> my_list.print_sorted()
[1]
>>> li3 = [3, 6, 2, 6, 1, 3]
>>> my_list = MyList(li3)
>>> my_list.print_sorted()
[1, 2, 3, 3, 6, 6]
>>> MyList = __import__('1-my_list').MyList
>>> my_list2.print_sorted()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'my_list2' is not defined

